cirq/google/api/v1/programs.py

```diff
@@ -345,6 +352,15 @@ def raise_missing_fields(gate_name: str):
     raise ValueError('invalid operation: {}'.format(proto_dict))
 
 
+def _qubit_from_proto_dict(proto_dict):
+    """Proto dict must have 'row' and 'col' keys."""
+    # TODO: Deprecate v1 proto method.
```

Comment:
Can we link to a person or issue here?

Reply:
I'm not sure the todo makes any sense.  We do want to deprecate v1 protos.  But that's a much bigger task than this single location.  Removed.